src/java/org/apache/cassandra/schema/SchemaKeyspace.java

```diff
@@ -1318,8 +1317,7 @@ static Set<String> affectedKeyspaces(Collection<Mutation> mutations)
     static void applyChanges(Collection<Mutation> mutations)
     {
         mutations.forEach(Mutation::apply);
-        if (SchemaKeyspace.FLUSH_SCHEMA_TABLES)
-            SchemaKeyspace.flush();
+        SchemaKeyspace.flush();
```

Comment:
I don't understand this change.
The comment above about `UNSAFE_SYSTEM` says:

```
** Disables flush changes to local and schema keyspaces.
```

but now we again flush the schema keyspace each time, as in prod?

Reply:
because `unsafeSystem` is taken into account in the `flush` method